kyo-core/shared/src/main/scala/kyo/fibers.scala

```diff
@@ -124,14 +124,12 @@ object Fibers extends Joins[Fibers] with fibersPlatformSpecific:
 
     type Effects = FiberGets & IOs
 
-    case object Interrupted
-        extends RuntimeException("Fiber Interrupted")
+    case class Interrupted(origin: Trace)
+        extends RuntimeException(s"Fiber Interrupted at ${origin}")
         with NoStackTrace:
         override def getCause() = null
```

Comment:
For later: Can we make a `KyoNoStackTrace(message: String)` abstract class? that way you won't need to extend multiple items or override a member.

Reply:
yeah, maybe we should always have the `Trace` as well to make debugging easier.